articles/healthcare-apis/find-identity-object-ids.md

```diff
@@ -25,7 +25,7 @@ $(Get-AzureADUser -Filter "UserPrincipalName eq 'myuser@consoso.com'").ObjectId
 or you can use the Azure CLI:
 
 ```azurecli-interactive
-az ad user show --upn-or-object-id myuser@consoso.com | jq -r .objectId
+az ad user show --upn-or-object-id myuser@consoso.com --query objectId --out tsv
```

Comment:
I think the latest versions of Azure CLI uses just `--id` instead of `--upn-or-object-id`, can you validate that works for you and correct at the same time?

Reply:
Yes, indeed, there's a deprecation warning with the `--upn-or-object-id` argument:

![Screenshot showing deprecation warning of --upn-or-object-id argument](https://user-images.githubusercontent.com/1086421/86791770-e0794680-c037-11ea-8bc9-7df6d38ef613.png)

I updated this to `--id` in 4a7e688:

![Screenshot showing non-deprecated usage of --id argument](https://user-images.githubusercontent.com/1086421/86792506-8f1d8700-c038-11ea-9998-ec0f4d65837c.png)